src/Extensions/Logging/ILogEventSender.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// Event source to send log messages
+	/// </summary>
+	public interface ILogEventSender
+	{
+		/// <summary>
+		/// Is logging enabled for specified log level
+		/// </summary>
+		bool IsEnabled(LogLevel level);
+
+
+		/// <summary>
+		/// Should this message be replayed in case of failure
+		/// </summary>
+		bool IsReplayableMessage(LogLevel logLevel);
+
+
+		/// <summary>Log message</summary>
```

Comment:
Nitpick: `<summary>` tags should be on their own lines

Reply:
I will resolve it in the follow-up pr https://github.com/microsoft/Omex/pull/168